### PR TITLE
fix: updating my profile

### PR DESCRIPTION
**Pull request by abdussalam02**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/858"></a>

### CODE REVIEW

**Review by schmelto — CHANGES_REQUESTED** [review]
Lgtm could you resolve the merge conflicts

File: public/data/Abdus_Salam.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "milestones": [
+    {
+      "title": "Started Tech Journey",
+      "date": "January 2018",
+      "icon": "books",
+      "color": "grey",
+      "description": "Started my Tech Journey at 16 and then left it for a year, but now I won't without making a mark"
+    }
+  ]
+}
```
Comment (Pradumnasaraf) [inline public/data/Abdus_Salam.json:42]:
```suggestion
}

```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

File: public/data/Abdus_Salam.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Abdus Salam",
+  "type": "personal",
+  "bio": "Hi there!, I am Python Developer and doing my best to contribute to open source. I'm not limited to Python only, I can do more fun things! (it's a long list) Connect with me and find out😉",
```
Comment (krishguptadev) [inline public/data/Abdus_Salam.json:4]:
```suggestion
  "bio": "Hi there! I am Python Developer and doing my best to contribute to open source. I'm not limited to Python only, I can do more fun things (it's a long list)! Connect with me and find out😉",
```

**Review by abdussalam02 — COMMENTED** [review]
committed the changes

File: public/data/Abdus_Salam.json [hunk 3]
```diff
@@ -16,13 +16,12 @@
...
       "name": "I'm on Telegram most of the time",
-      "url": "t.me/aladdin_02",
+      "url": "https://t.me/aladdin_02",
       "icon": "telegram"
     },
     {
       "name": "See my awesome tweets on Twitter",
-      "url": "https://twitter.com/abdussalam02",
-      "icon": "twitter"
+      "url": "https://twitter.com/abdussalam_02",
```
Comment (nathgoutam93) [inline public/data/Abdus_Salam.json:24]: Why you removed the icon ?
Also the GitHub actions complaining on the trailing comma you left after the URL as you removed the icon. Fix that

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by Pradumnasaraf — APPROVED** [review]
LGTM :)

**Pull request merged**